### PR TITLE
W-10877164 back up restore clarification

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
+Data about the state of Runtime Fabric is distributed across the Kubernetes cluster. To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup on external storage outside the Runtime Fabric cluster.
+
+For Runtime Fabric on VMs / Bare Metal, the back up and restore process includes backing up and restoring appliance-related custom resources such as logforwarder and smtp. The back up and restore process does not include backing up metrics data stored in InfluxDB that is generated by the applications.
+
+== When to Use the Back Up and Restore Process
+
 Common scenarios for using the back up and restore process include: 
 
 * Performing a major infrastructure upgrade, such as migrating nodes in a Runtime Fabric cluster to a new version of an operating system.
 * Upgrading Runtime Fabric for VMs / Bare Metal. In some cases, you may want to back up your cluster, install a new cluster using the new Runtime Fabric version, and then restore your backup on the new cluster.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-backup-restore.adoc]: @kmendiratta-crm does this contradict the new warning? Should we clarify here that you can't run them concurrently? Or is there a difference between BYOK and appliance?

Comment (IsaacEldridge) [reply modules/ROOT/pages/manage-backup-restore.adoc]: should I move the warning here?

Comment (hegderohit89) [reply modules/ROOT/pages/manage-backup-restore.adoc]: @sarjeet2013

Comment (hegderohit89) [reply modules/ROOT/pages/manage-backup-restore.adoc]: In my opinion its not confusing but may be in the warning we can explicitly include this:
In the warning section:
`not back up Runtime Fabric A, use that back up to create a new Runtime Fabric B while the Runtime Fabric A is till active,`.

OR

`Upgrading Runtime Fabric for VMs / Bare Metal. In some cases, you may want to back up your cluster, install a new cluster using the new Runtime Fabric version ON THE SAME CLUSTER, and then restore your backup on the new cluster.`

Comment (IsaacEldridge) [reply modules/ROOT/pages/manage-backup-restore.adoc]: @hegderohit89 I actually took both of these suggestions. Do you want to approve?

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 2]
```diff
@@ -2,17 +2,23 @@
...
 * Migrating your Runtime Fabric cluster from on-premises to the cloud or migrating from one cloud provider to another.
 * Performing a failover to a standby cluster. 
 
-Data about the state of Runtime Fabric is distributed across the Kubernetes cluster. To preserve applications in the event of system failure, confirm that Runtime Fabric is automatically backed up. When scheduling backups, follow best practices by configuring hourly backups. Store your backup on external storage outside the Runtime Fabric cluster.
-
-For Runtime Fabric on VMs / Bare Metal, the back up and restore process includes backing up and restoring appliance-related custom resources such as logforwarder and smtp. The back up and restore process does not include backing up metrics data stored in the influxdb that is generated by the applications.
-
+[WARNING]
+--
+Do not use the back up and restore process to create concurrent running instances of Runtime Fabric. In other words, do not back up Runtime Fabric A, use that back up to create a new Runtime Fabric B while Runtime Fabric A is till active, and attempt to run them concurrently. This creates application deployment issues.
```
Comment (kmendiratta-crm) [inline modules/ROOT/pages/manage-backup-restore.adoc]: typo: _while Runtime Fabric A is still active_

Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-backup-restore.adoc]:
```suggestion
Do not use the back up and restore process to create concurrent running instances of Runtime Fabric. In other words, do not back up Runtime Fabric A, use that back up to create a new Runtime Fabric B while Runtime Fabric A is still active, and attempt to run them concurrently. This creates application deployment issues.
```

**Review by kmendiratta-crm — DISMISSED** [review]

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 3]
```diff
@@ -76,19 +82,14 @@ When restoring on an existing Runtime Fabric cluster, use the same version of th
 scp /backup-path/to-restore.tar.gz your_username@remotehost:/opt/anypoint/runtimefabric/
 ----
 
-. To restore on a newly created cluster:
-.. Scale down all Runtime Fabric components on the original, backed-up cluster. 
-.. Confirm your Kubernetes (kubectl) context is set to the backed-up cluster. 
-.. Run the following command:
+
+. Confirm your Kubernetes (kubectl) context is set to the backed-up cluster, and scale down all Runtime Fabric components on the original, backed-up cluster:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-backup-restore.adoc]:
```suggestion
. Confirm your Kubernetes (kubectl) context is set to the backed-up cluster, and scale down all Runtime Fabric components on the original backed-up cluster:
```

**Review by fermujica — APPROVED** [review]

**Pull request merged**